src/main/java/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -256,26 +256,21 @@ public void loadDefaultGame(final boolean forceFactoryDefault) {
     // was using running a game within its root folder, we shouldn't open it)
     NewGameChooserEntry selectedGame;
     final String user = ClientFileSystemHelper.getUserRootFolder().toURI().toString();
-    if (!forceFactoryDefault && userPreferredDefaultGameUri != null && userPreferredDefaultGameUri.length() > 0
+    if (!forceFactoryDefault && userPreferredDefaultGameUri.length() > 0
         && userPreferredDefaultGameUri.contains(user)) {
       // if the user has a preferred URI, then we load it, and don't bother parsing or doing anything with the whole
       // game model list
       try {
         final URI defaultUri = new URI(userPreferredDefaultGameUri);
         selectedGame = new NewGameChooserEntry(defaultUri);
       } catch (final Exception e) {
-        selectedGame = selectByName(forceFactoryDefault);
+        selectedGame = selectByName(false);
         if (selectedGame == null) {
           return;
         }
       }
       if (!selectedGame.isGameDataLoaded()) {
-        try {
-          selectedGame.fullyParseGameData();
-        } catch (final GameParseException e) {
-          loadDefaultGame(true);
-          return;
-        }
+        selectedGame.fullyParseGameData();
```

Comment:
Not sure I understand what was going on in this hunk and the one below.  Do you mind elaborating on why the error handling is no longer necessary?

Reply:
In the context of the other branch this fit, but moved independently not quite as much. The error handling here is still very terrible, we silently handle the error and load the default game. No indication of why the game crashed, or that it even crashed, nor that the map selection changed.

This is the one thing I'm currently actively working for this branch to really improve. We should be showing an error message at least.